ruff is causing PR checks to fail

PR checks are failing on "Run make check" due to this error:

```
ruff .
Error: codespell_lib/_codespell.py:194:17: PLE1300 Unsupported format character '}'
make: *** [Makefile:42: ruff] Error 1
Error: Process completed with exit code 2.
```

Recently, the ruff version increased from `ruff-0.0.282` to `ruff-0.0.283`. Either fix the Python code, or downgrade ruff.

--- a/codespell_lib/_codespell.py
+++ b/codespell_lib/_codespell.py
@@ -190,10 +190,7 @@ class Summary:
         keys.sort()
 
         return "\n".join(
-            [
-                "{0}{1:{width}}".format(key, self.summary.get(key), width=15 - len(key))
-                for key in keys
-            ]
+            [f"{key}{self.summary.get(key):{15 - len(key)}}" for key in keys]
         )
 
 

